content/patterns/listbox/examples/listbox-collapsible.html

```diff
@@ -254,14 +267,13 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
             </tr>
             <tr data-test-id="listbox-aria-activedescendant">
               <td></td>
-              <th scope="row"><code>aria-activedescendant=&quot;ID_REF&quot;</code></th>
+              <th scope="row"><code>aria-activedescendant="ID_REF"</code></th>
               <td><code>ul</code></td>
               <td>
                 <ul>
-                  <li>Set by the JavaScript when it displays and sets focus on the listbox; otherwise is not present.</li>
-                  <li>Refers to the option in the listbox that is visually indicated as having keyboard focus.</li>
+                  <li>When an option in the listbox is visually indicated as having keyboard focus, refers to that option.</li>
```

Comment:
```suggestion
                  <li>When an option in the listbox is visually indicated as having keyboard focus, it refers to that option.</li>
```

Reply:
Same as with keyboard commands, for brevity, we avoid repetitiousness by leaving the subject out of the description.